src/battle_message.c

```diff
@@ -3074,7 +3075,7 @@ static const u8 *BattleStringGetPlayerName(u8 *text, u8 battlerId)
             toCpy = gSaveBlock2Ptr->playerName;
         break;
     case B_POSITION_PLAYER_RIGHT:
-        if ((gBattleTypeFlags & BATTLE_TYPE_RECORDED) && !(gBattleTypeFlags & (BATTLE_TYPE_MULTI | BATTLE_TYPE_INGAME_PARTNER)))
+        if (gTestRunnerEnabled)
```

Comment:
Should this keep the recorded battle check and be `|| gTestRunnerEnabled?` otherwise aren't you breaking opponent right trainer names in recorded battles

Reply:
You're right, let me fix that

Reply:
Done.